hubspot_sync/task_helpers.py

```diff
@@ -0,0 +1,38 @@
+""" Task helper functions for ecommerce """
+from django.conf import settings
+
+from hubspot_sync import tasks
+
+
+def sync_hubspot_user(user):
+    """
+    Trigger celery task to sync a User to Hubspot
+
+    Args:
+        user (User): The user to sync
+    """
+    if settings.MITOL_HUBSPOT_API_PRIVATE_TOKEN:
```

Comment:
This would cause lots of errors locally when running without a token (and I think dev instances should typically be run without hubspot integration enabled).

Reply:
I agree that would be prone to many sentry events.